WordPress/src/main/res/values/strings.xml

```diff
@@ -4244,4 +4244,5 @@ translators: %s: Select control option value e.g: "Auto, 25%". -->
     <string name="jp_migration_welcome_site_found_message">We found your site. Continue to transfer all your data and sign in to Jetpack automatically.</string>
     <string name="jp_migration_continue_button">Continue</string>
     <string name="jp_migration_help_button">Need help?</string>
+
```

Comment:
Nitpick: We could remove the empty line addition :)

Reply:
We can do it, although in fact this was just bringing it back after [it was removed](https://github.com/wordpress-mobile/WordPress-Android/pull/17387/files#diff-6bbab92723bb83a1300e86d496d62b9475a6a1e8b5a5feca27721edfbb09943fR4239-R4240) in #17387

I just checked both on `trunk` before the changes for the Welcome Screen were merged and on `release/21.1` and it seems to be conventionally present.

I was thinking it's best to keep this as it was before the changes, wdyt?